Libraries/Utilities/JsonHelpers.cs

```diff
@@ -22,5 +23,12 @@ public static bool ValidateJsonIfNotNull(string json)
 
             return true;
         }
+
+        /// <summary>
+        /// Returns a raw HTML string that's been JSON serialized and therefore safe to use within <c>&lt;script&gt;</c>
+        /// tags in a Razor view.
+        /// </summary>
+        public static IHtmlContent JsonHtmlContent(this string htmlString) =>

```

Comment:
Miért nem ennek is `Json` simán a neve? Másrészt helper classokba nem szoktunk extension methodokat, szal akkor legyen inkább külön egy `JsonStringExtensions`.

Reply:
Ez kimaradt.

Reply:
Tényleg. De azért nem simán Json, mert az eléggé félrevezetőnek érzem mivel egy `string.Json()` nem igazán `IHtmlContent` specifikus míg `LocalizedHtmlString`-et tényleg csak view-kban használsz.

Reply:
Aha, jó.